src/runtime/entrypoints/main.ts

```diff
@@ -1045,12 +1045,37 @@ addEventListener("popstate", async (e) => {
 document.addEventListener("submit", async (e) => {
   const el = e.target;
   if (el !== null && el instanceof HTMLFormElement && !e.defaultPrevented) {
-    const partial = el.getAttribute(PARTIAL_ATTR);
-    if (partial !== null) {
+    if (!checkClientNavEnabled()) {
```

Comment:
Very minor point, but could you make this check right at the start of the event handler instead?

Reply:
Ah no sorry, looks like it actually needs `el` as an argument